plugins/services/src/js/components/forms/VolumeMountFormSection.js

```diff
@@ -0,0 +1,124 @@
+import React, {Component} from 'react';
+import Objektiv from 'objektiv';
+
+import FieldError from '../../../../../../src/js/components/form/FieldError';
+import FieldInput from '../../../../../../src/js/components/form/FieldInput';
+import FieldLabel from '../../../../../../src/js/components/form/FieldLabel';
+import FormGroup from '../../../../../../src/js/components/form/FormGroup';
+import FormGroupContainer from '../../../../../../src/js/components/form/FormGroupContainer';
+import {FormReducer as volumeMounts} from '../../reducers/serviceForm/VolumeMounts';
+
+const errorsLens = Objektiv.attr('container', {}).attr('volumes', []);
+
+class VolumesFormSection extends Component {
```

Comment:
This again seems very similar to, but much more advanced than `VolumesFormSection` (it's even still named the same).
Could we not avoid this duplication, and create small section parts that they both can use?